core/src/test/java/kafka/test/ClusterTestExtensionsTest.java

```diff
@@ -236,6 +236,37 @@ public void testNotSupportedNewGroupProtocols(ClusterInstance clusterInstance) {
         Assertions.assertEquals(1, clusterInstance.supportedGroupProtocols().size());
     }
 
+
+
+    @ClusterTest(types = {Type.ZK, Type.CO_KRAFT, Type.KRAFT}, brokers = 3)
+    public void testCreateTopic(ClusterInstance clusterInstance) throws Exception {
+        String topicName = "test";
+        int partitions = 3;
+        short replicas = 3;
+        clusterInstance.createTopic(topicName, partitions, replicas);
+        clusterInstance.waitForTopic(topicName, partitions);
+
+        try (Admin admin = clusterInstance.createAdminClient()) {
+            Assertions.assertTrue(admin.listTopics().listings().get().stream().anyMatch(s -> s.name().equals(topicName)));
+        }
+    }
+
+    @ClusterTest(types = {Type.ZK, Type.CO_KRAFT, Type.KRAFT}, brokers = 3)
+    public void testWaitForMetadataSync(ClusterInstance clusterInstance) throws Exception {
+        String topicName = "test";
+        int partitions = 3;
+        short replicas = 3;
+        clusterInstance.createTopic(topicName, partitions, replicas);
+        clusterInstance.waitForTopic(topicName, partitions);
+
+        long offset = clusterInstance.waitForMeatdataSync();
```

Comment:
Could you please write test to make sure "it works well" rather than checking the return value. For example, the method is used to "wait" the metadata. Hence, we should make some metadata changes and then call `waitForMetadataSync`. Finally, we call the APIs using the metadata to check the metadata is synced.